### PR TITLE
clock created

### PR DESCRIPTION
**Pull request by pr1syazhniy**
[DEMO LINK](https://pr1syazhniy.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/stopwatch.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.clock {
+  width: $clock-size;
+  height: $clock-size;
+
+  @include position-value;
+  position: absolute;
```
Comment (vlad-demchuk) [inline src/styles/stopwatch.scss:6]: image

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
it would be better to create separate folders for blocks and sass utils
image

File: src/styles/stopwatch.scss (hunk 1 above)
Comment (pr1syazhniy) [reply src/styles/stopwatch.scss:6]: yeah,forgot about that)

Comment (vlad-demchuk) [reply src/styles/stopwatch.scss:6]: not fixed https://github.com/mate-academy/layout_stop-watch/pull/2321/files#r1169569301
you should move external styles to bem element. you can use mix(refer to checklist examples)

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="container">
+      <div class="clock">
```
Comment (vlad-demchuk) [inline src/index.html:16]: you have used the bad example)
image

File: src/styles/blocks/stopwatch.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.container {
+  @include position-value;
+  position: absolute;
+}
```
Comment (vlad-demchuk) [inline src/styles/blocks/stopwatch.scss:4]: follow checklist 
image

make parent element of clock a bem block and make clock a bem element
don't forget to creaet a separate file for created bem block

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**